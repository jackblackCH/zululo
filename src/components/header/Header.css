.c-header {
  font-size: 30px;
  margin: 10px;
  letter-spacing: 3.5px;
  font-weight: 200; }

.c-header__logo {
  width: 50px; }
