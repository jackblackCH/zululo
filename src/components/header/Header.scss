.c-header {
    font-size: 30px;
    margin: 10px;
    letter-spacing: 3.5px;
    font-weight: 200;

    &__logo {
      width: 50px;
    }
}
