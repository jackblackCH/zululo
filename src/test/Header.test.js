import Header from '../components/header';
import renderer from 'react-test-renderer';
import React from 'react';


describe('Component: Header', () => {

  it('should render the Header', () => {
    const tree = renderer.create(
      <Header></Header>
    ).toJSON();
    expect(tree).toMatchSnapshot()
  })

  it('should have a className', () => {
    const tree = renderer.create(
      <Header className='c-header'></Header>
    ).toJSON();
    expect(tree).toMatchSnapshot()
  })

})
