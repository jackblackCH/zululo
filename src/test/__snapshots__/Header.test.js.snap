exports[`Component: Header should have a className 1`] = `
<header
  className="c-header">
  <img
    className="c-header__logo"
    src="/assets/logo.png" />
  Zululo
</header>
`;

exports[`Component: Header should render the Header 1`] = `
<header
  className="c-header">
  <img
    className="c-header__logo"
    src="/assets/logo.png" />
  Zululo
</header>
`;
